images/win/toolsets/toolset-2019.json

```diff
@@ -163,9 +163,12 @@
             "cmake;3.6.4111459",
             "cmake;3.10.2.4988404",
             "patcher;v4",
-            "ndk;21.3.6528147",
             "cmdline-tools;latest"
-        ]
+        ],
+        "ndk": {
+            "lts": "21.3.6528147",
```

Comment:
Can we specify only the major version here and install the latest minor for it? Because lts can be updated through the year.

Reply:
Looks like Android sdkmanager requires full versions of packages for installation and it doesn't support wildcards. I've tried to install it with only major version or wildcard and it throws error:
```
Failed to find package ndk;22
```
or
```
Failed to find package ndk;22.*
```

Reply:
Yeah we need additional logic in the installation script to get all available versions and take the latest minor one

Reply:
Done!